app/Http/Controllers/IncomingTextController.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\TwilioRequest as Request;
+
+class IncomingTextController extends Controller
+{
+    protected $commands = [
+        "unsubscribe" => [
+            "STOP",
+            "STOPAll",
+            "UNSUBSCRIBE",
+            "CANCEL",
+            "END",
+            "QUIT",
+        ],
+        "subscribe" => [
+            "START",
+            "YES",
+        ]
+    ];
+
+    /**
+     * Starting point for all incoming text messages.
+     */
+    public function receiveText(Request $request)
+    {
```

Comment:
What if we did this?

```php
if ($this->commandExists($request)) {
    return $this->commandForRequest($request);
}

return $this->responseMessage('...');
```

Then those methods can be responsible for taking this syntax below and making it function in the above, little bit more readable syntax